### PR TITLE
Throws Error when attempting to define more than one default display

### PR DESCRIPTION
**Pull request by GranolaDaniel**
Fixes issue: https://github.com/missionpinball/mpf/issues/1315

### CODE REVIEW

File: mpfmc/uix/display.py [hunk 1]
```diff
@@ -139,14 +139,18 @@ def _display_created(self, *args) -> None:
...
 
-        # If this display is configured as the default, set it
+        # If this display is configured as the default, set it.
+        # If this display would overwrite an existing default, raise an AssertionError.
         try:
             if self.config['default']:
-                self.mc.targets['default'] = self
+                if 'default' not in self.mc.targets:
+                    self.mc.targets['default'] = self
+                elif 'default' in self.mc.targets:
```
Comment (jabdoa2) [inline mpfmc/uix/display.py]: This can be simplified. Since the condition of if and elif is the same just use "else".

File: mpfmc/uix/display.py [hunk 2]
```diff
@@ -139,14 +139,18 @@ def _display_created(self, *args) -> None:
             self.size = self.native_size
             Clock.schedule_once(self._display_created, 0)
             return
-
+
```
Comment (jabdoa2) [inline mpfmc/uix/display.py]: Please remove those space characters. They make the linter unhappy:

> Messages
> ========
> mpfmc/uix/display.py
>   Line: 142
>     pep8: W293 / blank line contains whitespace (col 1)
>   Line: 153
>     pep8: E501 / line too long (218 > 120 characters) (col 121)

File: mpfmc/uix/display.py [hunk 3]
```diff
@@ -139,14 +139,18 @@ def _display_created(self, *args) -> None:
...
-        # If this display is configured as the default, set it
+        # If this display is configured as the default, set it.
+        # If this display would overwrite an existing default, raise an AssertionError.
         try:
             if self.config['default']:
-                self.mc.targets['default'] = self
+                if 'default' not in self.mc.targets:
+                    self.mc.targets['default'] = self
+                elif 'default' in self.mc.targets:
+                    raise AssertionError('Multiple displays have been set as the default. Please choose a single display to default to (\"{}\" is currently set as the default).'.format(self.mc.targets['default'].name))
```
Comment (jabdoa2) [inline mpfmc/uix/display.py]: Please break the line. More than 120 characters make the linter unhappy. https://travis-ci.org/missionpinball/mpf-mc/jobs/550076932#L5075

**Review by jabdoa2 — CHANGES_REQUESTED** [review]
Thanks for your effort! Looks good. Just some minor nitpicks.

**Comment by jabdoa2** [comment]
One more:
```
Messages
========
mpfmc/uix/display.py
  Line: None
    pep8: E101 / indentation contains mixed spaces and tabs (col 1)
  Line: 153
    pep8: W191 / indentation contains tabs (col 1)
```

I can also fix that later if you want.

**Review by jabdoa2 — APPROVED** [review]

**Comment by GranolaDaniel** [comment]
> One more:
> 
> ```
> Messages
> ========
> mpfmc/uix/display.py
>   Line: None
>     pep8: E101 / indentation contains mixed spaces and tabs (col 1)
>   Line: 153
>     pep8: W191 / indentation contains tabs (col 1)
> ```
> 
> I can also fix that later if you want.

I believe that I've switched that line back to spaced indentation. Please let me know if you have any feedback, and thank you for the guidance!

**Comment by jabdoa2** [comment]
Now it complains about long lines again:

 mpfmc/uix/display.py Line: 153 pep8: E501 / line too long (129 > 120 characters) (col 121)

**Comment by jabdoa2** [comment]
Now it complains about long lines again:

 mpfmc/uix/display.py Line: 153 pep8: E501 / line too long (129 > 120 characters) (col 121)

File: mpfmc/uix/display.py [hunk 4]
```diff
@@ -139,14 +139,19 @@ def _display_created(self, *args) -> None:
...
+        # If this display is configured as the default, set it.
+        # If this display would overwrite an existing default, raise an AssertionError.
         try:
             if self.config['default']:
-                self.mc.targets['default'] = self
+                if 'default' not in self.mc.targets:
+                    self.mc.targets['default'] = self
+                else:
+                    raise AssertionError('Multiple displays have been set as the default. Please choose a single \
+                        display to default to (\"{}\" is currently set as the default).'.format(
```
Comment (jabdoa2) [inline mpfmc/uix/display.py:153]: Python would like this one indented as much as the braket in the previous line. You can run the linter locally if you want. Installing it: pip install prospector. Running it inside the mpf folder: prospector

```
mpfmc/uix/display.py
  Line: 154
    pep8: E126 / continuation line over-indented for hanging indent (col 29)
```

Comment (GranolaDaniel) [reply mpfmc/uix/display.py:153]: Thanks for pointing me to the linter! I've just submitted a new PR after running it. Let me know if anything else looks off.

**Comment by jabdoa2** [comment]
Thanks for all the effort!

**Pull request merged**